plugins/resteasy-http/src/main/java/org/radargun/http/service/AbstractRESTEasyService.java

```diff
@@ -73,6 +81,10 @@ public synchronized void start() {
          return;
       }
 
+      if (trustAll) {
+         HttpsHelper.trustAll();
```

Comment:
Setting trustall globally as default if one of the clients sets it isn't a good design. But probably shouldn't be issue for RadarGun as you are probably not going to run performance tests against cluster with valid and invalid certificats.

Reply:
It is false by default

Reply:
I didn't mean the property itself but the fact that one client setting the property to true sets static variables in `HttpsURLConnection`:
```
HttpsURLConnection.setDefaultSSLSocketFactory(sc.getSocketFactory());
HttpsURLConnection.setDefaultHostnameVerifier(validHosts);
```